packages/storage/src/database/common/rollup-history.ts

```diff
@@ -1,11 +1,12 @@
-import { TRollUpHistory } from '@zkdb/common';
+import { TRollUpHistory, TRollUpHistoryRecord } from '@zkdb/common';
 import { ReplaceOptions, UpdateResult } from 'mongodb';
 import { zkDatabaseConstants } from '../../common/index.js';
 import { DB } from '../../helper/db-instance.js';
 import logger from '../../helper/logger.js';
 import ModelBasic from '../base/basic.js';
+import { WithoutId } from 'mongodb';
 
```

Comment:
Move to single line with import { ReplaceOptions, UpdateResult } from 'mongodb';